gems/aws-sdk-core/spec/aws/ini_parser_spec.rb

```diff
@@ -34,34 +34,40 @@ module Aws
 s3 =
    endpoint_url = https://localhost:8000
 
+[profile blank-property]
+aws_session_token = 
+s3 = 
+   region = ap-southeast-1
```

Comment:
Does this also work with a blank property inside s3? Maybe we should add one here.

Reply:
It does!  Added one here + expect in the test.